docassemble/ALDashboard/data/questions/install_assembly_line.yml

```diff
@@ -198,31 +218,111 @@ fields:
     hide if: skip_efile
     required: False
 ---
-id: feedback form config
+# This must be tested with a private repository
+#  If your GitHub organization's repositories are private, the account must have permission to see it and file issues.
+id: github PAT
 continue button field: feedback_form_config
 question: |
-  GitHub feedback and about information
+  Feedback and "About" page information with GitHub
 subquestion: |
-  You should configure a dedicated GitHub account with permission to make
-  pull requests on public repositories and generate a private access token
-  for it.
+  Assembly Line uses GitHub to let you:
+  
+  1. Get user feedback about each interview.
+  2. Display a "last updated" date on the "About" page of an interview.
+
+  First, in order to use GitHub you should create a dedicated GitHub account. You can use an account that already exists, but we recommend having an account that only handles Assembly Line functionality.
+  
+  The account must have permission make pull requests on your GitHub organization's repositories. You can read [GitHub's documentation about setting an account's permissions](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization).
   
-  This feature is used to add feedback to production interviews as well
-  as to display a "last updated" date on the about page.
+  Give us the information about that account and set up its permissions.
 fields:
+  - What is the username of the account: the_config['github issues']['username']
+    default: ${ the_config['github issues'].get('username', '') }
+    help: |
+      This will set the `config['github issues']['username']` and the `config['github readonly']['username']` values.
```

Comment:
This flat representation isn't really like the representation in the YAML. I think it's better to leave it out.